js/core/dom_component.js

```diff
@@ -59,6 +59,8 @@ const DOMComponent = Component.inherit({
     _visibilityChanged: abstract,
     _dimensionChanged: abstract,
 
+    _customClass: null,
```

Comment:
I think that default value for _customClass field should be assigned in the DOMComponent constructor.